runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -53,11 +53,14 @@ jvmtiGetStackTrace(jvmtiEnv* env,
 		vmFuncs->internalEnterVMFromJNI(currentThread);
 
 		ENSURE_PHASE_LIVE(env);
-
 		ENSURE_NON_NEGATIVE(max_frame_count);
 		ENSURE_NON_NULL(frame_buffer);
 		ENSURE_NON_NULL(count_ptr);
 
+		if (NULL != thread) {
```

Comment:
Might be nice to add a new macro for this since it comes up many times. `ENSURE_JTHREAD_ALLOW_NULL` perhaps.